ntoskrnl/ex/time.c

```diff
@@ -347,13 +387,15 @@ ExpSetTimeZoneInformation(PRTL_TIME_ZONE_INFORMATION TimeZoneInformation)
     HalQueryRealTimeClock(&TimeFields);
     RtlTimeFieldsToTime(&TimeFields, &LocalTime);
 
-    /* FIXME: Calculate transition dates */
```

Comment:
And what's that FIXME about?

Reply:
This has been in the code since Eric Kohl originally wrote this section of the code on 12/1/2004 in R11885 and commit 019c9076 [019c9076eeb2d3696c20160424d4a86a5430efc6]. I believe that he placed it here as a placeholder to note that he was not calculating the Daylight Saving Time Start and Standard Time Start times and accounting for them. I removed the comment, because with this new revision these are not taken into account. Thanks.

Reply:
> on 12/1/2004 in R11885 and commit [019c907](https://github.com/reactos/reactos/commit/019c9076eeb2d3696c20160424d4a86a5430efc6)

Rather df74513933549485017561ea18e551c8ddcf8c93 (r11867).

Reply:
Sorry, I looked at the left side of the diff instead of the right. @SergeGautherie, you are correct. Thanks.